core/src/avm2/globals/flash/display/bitmap_data.rs

```diff
@@ -439,15 +439,58 @@ pub fn set_pixels<'gc>(
             .ok_or("ArgumentError: Parameter must be a bytearray")?;
 
         operations::set_pixels_from_byte_array(
-            activation.context.gc_context,
+            activation,
             bitmap_data,
             x,
             y,
             width,
             height,
             &mut ba_write,
-        )
-        .map_err(|e| e.to_avm(activation))?;
+        )?;
+    }
+
+    Ok(Value::Undefined)
+}
+
+/// Implements `BitmapData.setPixels`.
+pub fn set_vector<'gc>(
+    activation: &mut Activation<'_, 'gc>,
+    this: Option<Object<'gc>>,
+    args: &[Value<'gc>],
+) -> Result<Value<'gc>, Error<'gc>> {
+    let rectangle = args.get_object(activation, 0, "rect")?;
+
+    let vector = args
```

Comment:
From #10275 this should use `get_object` (which has a null check), and the argument name should be `imputVector` instead of `inputVector` for consistency with a Flash typo.